Makefile

```diff
@@ -1,16 +1,16 @@
 GOMOBILE=gomobile
 GOBIND=$(GOMOBILE) bind
 BUILDDIR=$(shell pwd)/build
-IOS_ARTIFACT=$(BUILDDIR)/WaterMob.framework
+IOS_ARTIFACT=$(BUILDDIR)/Watermob.xcframework
 ANDROID_ARTIFACT=$(BUILDDIR)/watermob.aar
-IOS_TARGET=ios
+IOS_TARGET=ios,iossimulator
 ANDROID_TARGET=android
 ANDROID_API=27
 # LDFLAGS='-s -w -X google.golang.org/protobuf/reflect/protoregistry.conflictPolicy=warn'
 LDFLAGS='-s -w'
 IMPORT_PATH=github.com/gaukas/watermob
 
-BUILD_IOS="cd $(BUILDDIR) && $(GOBIND) -a -ldflags $(LDFLAGS) -target=$(IOS_TARGET) -o $(IOS_ARTIFACT) $(IMPORT_PATH)"
+BUILD_IOS="$(GOBIND) -target=$(IOS_TARGET) -o $(IOS_ARTIFACT)"
```

Comment:
Is this change at all necessary? (and by what means)

Reply:
Yeah, the previous command doesn't work for building iOS, so I changed to this. And I didn't add the `-ldflags` for now.

Reply:
If at all possible we should at least debug this to learn why it will not work. 

The original command `cd` into the build output dir at `$(BUILDDIR)` and builds the source in `$(IMPORT_PATH)` which should be resolved by `gomobile` automatically after identifying the package name. 

Meanwhile at this point we might want to make build command for Android and iOS consistent. So I will push a commit to update the Android command into the same style.

Reply:
Thanks for the alignment! Yeah, I'll try to see what happened with the previous command async so not block this PR to be merged.

Reply:
Thanks!